README.md

```diff
@@ -8,9 +8,18 @@
 In addition, the proxied web page does not know that it is opened under a proxy. The proxy intercepts access attempts to all URL-containing properties and provides the original values.
 
 ## First Look
-1. Clone the Hammerhead repository (`git clone https://github.com/DevExpress/testcafe-hammerhead.git`)
-2. Build it (`gulp run build`)
-3. Run the [Hammerhead playground](https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/playground/server.js) (`npm run http-playground`) to see our proxy in action.
+1. Clone the Hammerhead repository
+    ```
+    git clone https://github.com/DevExpress/testcafe-hammerhead.git
```

Comment:
This command clones the repository to the `testcafe-hammerhead` folder. If you run the `npm install` command after it, then you get message `ENOENT: no such file or directory <current_dir>\package.json`.

To prevent this, need to change `git clone...` command as follows:
`git clone https://github.com/DevExpress/testcafe-hammerhead.git .`